README.md

```diff
@@ -79,3 +79,119 @@
     ```
     python flaskr/manage.py shell
     ```
+
+### Accessing GraphQL Endpoint
```

Comment:
This is really helpful.  Especially to have the expected output.  Thanks for including.